File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.js
function solution(numbers, direction) {
    var answer = [];
    if(direction === "right"){
        answer.push(numbers[numbers.length-1]);
        for(let i =0; i<numbers.length-1; i++){
            answer.push(numbers[i]);
        }
    }else{
        for(let i =0; i<numbers.length-1; i++){
            answer.push(numbers[i+1]);
        }
        answer.push(numbers[0]);
        
    }
    return answer;
}

File: 프로그래머스/0/120844. 배열 회전시키기/배열 회전시키기.py
def solution(numbers, direction):
    answer = [i for i in range(len(numbers))]
    for (idx,num) in enumerate(numbers):
        if direction == 'right':
            if idx == len(numbers)-1:
                answer[0] = num
            else:
                answer[idx+1] = num
        else:
            if idx == 0:
                answer[len(answer)-1] = num
            else:
                answer[idx-1] = num    
    return answer